li-apache-kafka-clients/src/main/java/com/linkedin/kafka/clients/consumer/LiKafkaConsumerConfig.java

```diff
@@ -90,6 +90,9 @@
 
   private static final String AUDITOR_CLASS_DOC = "The auditor class to use for the consumer";
 
+  private static final String ENABLE_SHALLOW_ITERATOR_DOC = "If true consumer will return the raw byte record to caller directly"
```

Comment:
Can you also add the original documentation, along with this statement?

Reply:
sure.